semantics/executable-spec/small-steps.cabal

```diff
@@ -34,19 +34,23 @@ library
                      , Data.CannonicalMaps
                      , Data.MemoBytes
                      , Data.Coders
+                     , Control.Provenance
                      , Control.Iterate.SetAlgebra
                      , Control.Iterate.Collect
                      , Control.SetAlgebra
   -- other-modules:
   -- other-extensions:
-  build-depends:       array
+  build-depends:       aeson
+                     , array
                      , base >=4.11 && <5
                      , bytestring
                      , cardano-prelude
                      , containers
                      , cryptonite
+                     , data-default-class
                      , deepseq
                      , free
+                     , hashmap
```

Comment:
I just noticed this dependency was added while compiling this repo. Hash maps/sets are a red flag, ask @dcoutts about them :smile:. However, it is apparently not even used? Please remove it then, so that no one else will get worried about it for nothing.

Moreover, `hashmap` was deprecated in favour of `unordered-containers`, so a double no no :slightly_smiling_face:.